### PR TITLE
Add new style for Population (French Edition)

### PR DESCRIPTION
**Pull request by nalimilan**
Supersedes https://github.com/citation-style-language/styles/pull/1360.

### CODE REVIEW

**Comment by adam3smith** [comment]
same question here--these styles differ in more than language?

On Tue, Jan 27, 2015 at 10:59 AM, Milan Bouchet-Valat <
notifications@github.com> wrote:

> Supersedes #1360
> <https://github.com/citation-style-language/styles/pull/1360>.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/citation-style-language/styles/pull/1361
> Commit Summary
>
>    - Add new styles for Population (French & English)
>
> File Changes
>
>    - *A* population-en.csl
>    <https://github.com/citation-style-language/styles/pull/1361/files#diff-0>
>    (257)
>    - *A* population.csl
>    <https://github.com/citation-style-language/styles/pull/1361/files#diff-1>
>    (262)
>
> Patch Links:
>
>    - https://github.com/citation-style-language/styles/pull/1361.patch
>    - https://github.com/citation-style-language/styles/pull/1361.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1361>.
>



-- 
Sebastian Karcher, PhD
Department of Political Science
Northwestern University

**Comment by nalimilan** [comment]
...and basically same reply as for the other PR.

**Comment by nalimilan** [comment]
Given the discussion at #1360, I've updated the PR to only add the French version for now.

**Comment by rmzelle** [comment]
The documentation link needs to be updated. I guess "http://www.journal-population.com/submission-guidelines/" or "http://www.revue-population.fr/soumettre-un-article/"?

**Comment by nalimilan** [comment]
Right, I've also added the EISSN.

**Comment by rmzelle** [comment]
Isn't the eISSN 1957-7966? See http://www.cairn.info/revue-population.htm
(and we have the `<eissn/>` element for the eISSN)

It's easier for us if you don't change your original commit, by the way, since I can't see what you changed now. It's fine to have a few commits per pull request.

**Comment by nalimilan** [comment]
Done. Actually I had pushed exactly the same commit without any changes. I've added another one.

**Comment by nalimilan** [comment]
And here's the English dependent style (for now "Ph.D. thesis" is not translated, not a big deal). I can squash the commits if you want.

File: dependent/population-en.csl [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US" page-range-format="expanded" demote-non-dropping-particle="never">
```
Comment (rmzelle) [inline dependent/population-en.csl]: @nalimilan, can you remove the `class="in-text"` and `page-range-format="expanded" demote-non-dropping-particle="never"` bits here? They don't do anything in dependent styles.

Comment (nalimilan) [inline dependent/population-en.csl]: Done. Indeed no need to repeat these items.

**Comment by rmzelle** [comment]
Great, thanks!

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**